code/index.html

```diff
@@ -1,22 +1,120 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" sizes="16x16" href="favicon.png">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Espresso Crema</title>
+</head>
+<body>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+<!-- HEADER-->    
+<header href="#home">
+ <div class="hamburger">
+    <div></div>
+    <div></div>
+    <div></div>
+ </div>
+<a href="#home"><img src="logo.png" alt="logo"></a>
+<nav>
+    <a href="#blog">Blog</a>
+    <a href="#gallery">Gallery</a>
+    <a href="#contact">Contact</a>
+</nav> 
+</header>
 
-    <section class="">
-      <!-- The other news  -->
+<!-- MAIN ARTICLE-->
+<section class="main-article">
+ <a href="https://www.nbcnews.com/health/health-news/coffee-good-you-more-science-shows-n888356" target="_blank">
+    <section class="container">
+        <img src="girl-coffee.jpg" alt="girl-drinks-coffee">
```

Comment:
No need for dashes in the alt tag! And just a heads up, I think this image file is super big (it takes a few seconds to load fully).

Reply:
Ojj, sorry! Next time I am going to check it !